keps/sig-api-machinery/4006-transition-spdy-to-websockets/kep.yaml

```
@@ -0,0 +1,45 @@
+title: Transition from SPDY to Websockets
+kep-number: 4006
+authors:
+  - "@seans3"
+owning-sig: sig-api-machinery
+participating-sigs:
+  - sig-cli
+status: implementable
+creation-date: 2023-05-15
+reviewers:
+  - "@aojea"
+  - "@ardaguclu"
+  - "@soltysh"
+approvers:
+  - "@deads2k"
+  - "@jpbetz"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.28"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.28"
+  beta: "v1.29"
+  stable: "v1.30"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: CLIENT_REMOTE_COMMAND_WEBSOCKETS
```

Nit: 
```suggestion
  - name: KUBECTL_CLIENT_REMOTE_COMMAND_WEBSOCKETS
```

from what I just checked we prefix all our envs with `KUBECTL_`